orchestration/templates/ingest-hca.yaml

```diff
@@ -174,6 +174,10 @@ spec:
                 - name: bq-dataset
                   value: {{ $datasetName | quote }}
 
+          - name: move-error-logs-to-gcs
+            dependencies: [ingest-non-file-metadata, ingest-file-metadata, ingest-data-files]
```

Comment:
Added these dependencies here because I want this step to run at the very end of the ingest process (so we don't miss any logs).